IncSet.c

```diff
@@ -155,6 +157,28 @@ static bool IncMode_find(const IncMode* mode, Panel* panel, IncMode_GetPanelValu
    }
 }
 
+static int IncSet_lastCharLen(const char *s, int slen) {
+   if (slen < 1) {
+      return 0;
+   }
+
+   wchar_t w[INCMODE_MAX + 1];
+   size_t wlen = mbstowcs(w, s, slen);
+   if (wlen == (size_t)-1) {
+      return 1;
+   }
+
+   char mb[MB_LEN_MAX];
+   int clen = wcrtomb(mb, w[wlen - 1], NULL);
+   if (clen < 0) {
+      return 1;
+   } else if (clen > slen) {
+      clen = slen;
+   }
```

Comment:
`clen` should be of type `size_t`. It would be better if `slen` is also a `size_t` type.

Reply:
@Explorer09 Is this one resolved?

Reply:
It seems that the function has been reworked (to `IncSet_backspaceLen`) and no longer uses `wcrtomb`. I think I can close this part of the review as "no longer applicable".

Reply:
P.S. I mean, mark this as resolved. (I can't close this by myself).